Video: Goodbye Monada - Hello Wing Cloud (#565)
winglang/docsite: blog/2023-08-10-wingly-introducing-wing-cloud.md

---

> title: "Video: Goodbye Monada - Hello Wing Cloud"

Author: yes

authors: 
  - eyalk
  - shaib
tags: [wingly, winglang, video,wingcloud]
hide_table_of_contents: true
---

<iframe width="560" height="315" src="https://www.youtube.com/embed/mvWv9bVyQdc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>

> > In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round! Listen in as they discuss the Vision of Wing Cloud, and how it relates to our ongoing commitment to Winglang.

Author: ```suggestion
> In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round! Listen in as they discuss the vision of Wing Cloud, and how it relates to our ongoing commitment to Winglang as an open-source project.
```

> > In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round! Listen in as they discuss the vision of Wing Cloud, and how it relates to our ongoing commitment to Winglang as an open-source project.

Reviewer: ```suggestion
> In this short video, Eyal and Shai give some behind the scenes details of Wing Cloud’s recent launch out of stealth, with a $20 million seed round. Listen in as they discuss the vision of [Wing Cloud](https://wing.cloud) as well as our ongoing commitment to Winglang as an open-source project.
```
